scripts/globals/job_utils/ranger.lua

```diff
@@ -0,0 +1,284 @@
+-----------------------------------
+-- Ranger Job Utilities
+-----------------------------------
+require('scripts/globals/utils')
+-----------------------------------
+xi = xi or {}
+xi.job_utils = xi.job_utils or {}
+xi.job_utils.red_mage = xi.job_utils.red_mage or {}
```

Comment:
Utils name should be ranger

Reply:
fixed in latest commit: [3600528c2ecf40e6e8863a0015916a2e7e64defb](https://github.com/LandSandBoat/server/pull/5889/commits/3600528c2ecf40e6e8863a0015916a2e7e64defb)